src/transport_performance/gtfs/validation.py

```diff
@@ -17,7 +17,10 @@
 from typing import Union
 from plotly.graph_objects import Figure as PlotlyFigure
 
-from transport_performance.gtfs.routes import scrape_route_type_lookup
+from transport_performance.gtfs.routes import (
+    scrape_route_type_lookup,
+    get_saved_route_type_lookup,
+)
```

Comment:
Appreciate that I'm going to start pointing at something not within this diff, so happy to raise a separate issue. Though it should be a quick patch. 
```
from transport_performance.gtfs.report.report_utils import (
    TemplateHTML,
    _set_up_report_dir,
    GTFS_UNNEEDED_COLUMNS,
)
```
Better to store `GTFS_UNNEEDED_COLUMNS` as a class attribute within `TemplateHTML`.

Reply:
I don't object to this however could you explain the rationale of this for me?